### PR TITLE
Streamline publish process

### PR DESCRIPTION
**Pull request by HarelM**
## Launch Checklist

This aligns the build process with maplibre-gl where you open a PR with a version bump and then the package is published automatically after this PR is merged to main branch.

 - [x] Confirm **your changes do not include backports from Mapbox projects** (unless with compliant license) - if you are not sure about this, please ask!
 - [x] Briefly describe the changes in this PR.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -44,4 +44,11 @@ There are a few layers of tests - integration, unit and build tests.
 npm run test-build
 npm run test-integration
 npm run test-unit
-```
\ No newline at end of file
+```
+
+# Publish style-spec NPM package
```
Comment (louwers) [inline CONTRIBUTING.md:49]:
```suggestion
# Publish style-spec npm package
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -44,4 +44,11 @@ There are a few layers of tests - integration, unit and build tests.
...
+In order to publish the package to NPM:
```
Comment (louwers) [inline CONTRIBUTING.md:50]:
```suggestion
In order to publish the package to npm:
```

**Review by louwers — APPROVED** [review]
Convenient!

File: CONTRIBUTING.md (hunk 1 above)
Comment (HarelM) [reply CONTRIBUTING.md:49]: NPM stands for Node Package Manager, it should be with capitals I believe...

**Pull request merged**